configure.ac

```diff
@@ -301,6 +302,20 @@ AM_COND_IF([ENABLE_LARGE_TILES], [
     AC_DEFINE(CONFIG_LARGE_TILES, 0, [use small tiles])
 ])
 
+## Make a guess about the source code version
+AS_IF([test -d "${srcdir}/.git"], [
+    AS_IF([which git], [
```

Comment:
This should suppress standard output. It seems there’s no standard way to do this other than just `which git >/dev/null`.

(Alternatively, with more effort, this could prettify the output to look like an autoconf check.)

Reply:
Will `>/dev/null` work for Cygwin builds on Windows?

Reply:
(re: /dev/null) FWIW that’s exactly what I expected, and I fully expect it to work in both Cygwin and MSYS on Windows.

FYI:

Cygwin’s whole goal is to be a full POSIX environment on Windows, so I absolutely expect /dev/null to work there.

MSYS is a more lightweight thing that is mainly a POSIX shell, with an explicit goal to be [just] posixy enough to run build scripts for software originating from the posixland. Precisely for these reasons, I expect /dev/null to work there, too. (Although I use MSYS myself sometimes, my memory isn’t quite fresh enough to assert that I’ve “definitely seen” it work.)

And I imagine the configure script itself expects a POSIX-like shell—even if autoconf does cater for compatibility with exotic and ancient unixes, /dev/null seems like a fairly basic need. Which you’ve now confirmed by saying it does already use /dev/null.